src/App.vue

```diff
@@ -74,35 +67,34 @@ export default class App extends Vue {
         }
     }
 
-    body {
-        font-size: 2.25rem;
-    }
-
     .logo-container {
         width: 100%;
-        padding: 4rem;
+        padding: 4rem 3rem;
         box-sizing: border-box;
     }
 
     .logo {
         height: 3.625rem;
         box-sizing: border-box;
-        padding-left: 6rem;
+        /* padding-left: 6rem; */
         font-size: 2.125rem;
         font-weight: 600;
```

Comment:
Later you changed all numerical values for font-weight. Why not here?

Reply:
What's wrong with numerical values though? `normal` is 400, `bold` is 700. There is no name for 600.

Reply:
nothing wrong with them i just saw you changed it in another file.

Reply:
Just to keep in mind, adding a font-weight should be about 20kb extra to download.